config/src/main/java/com/typesafe/config/impl/ConfigNodeRoot.java

```diff
@@ -0,0 +1,38 @@
+package com.typesafe.config.impl;
+
+import com.typesafe.config.ConfigException;
+import com.typesafe.config.ConfigSyntax;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+final class ConfigNodeRoot extends ConfigNodeComplexValue {
+    ConfigNodeRoot(Collection<AbstractConfigNode> children) {
+        super(children);
+    }
+
+    protected ConfigNodeComplexValue value() {
+        for (AbstractConfigNode node : children) {
+            if (node instanceof ConfigNodeComplexValue) {
+                return (ConfigNodeComplexValue)node;
+            }
+        }
+        throw new ConfigException.BugOrBroken("ConfigNodeRoot did not contain a value");
```

Comment:
does the parser check for this and throw before creating the ConfigNodeRoot? If not we could reach this exception due to an empty config file, rather than due to a bug in the program. (I'm not sure a zero-byte config file is even disallowed - it may have been ok to do `parseString("")` before - I'd sort of expect it was, but I'm not sure I ever tried it)